client/network/src/block_request_handler.rs

```diff
@@ -62,7 +62,7 @@ pub fn generate_protocol_config(protocol_id: &ProtocolId) -> ProtocolConfig {
 		name: generate_protocol_name(protocol_id).into(),
 		max_request_size: 1024 * 1024,
 		max_response_size: 16 * 1024 * 1024,
-		request_timeout: Duration::from_secs(40),
+		request_timeout: Duration::from_secs(20),
```

Comment:
Why did you change the timeout?

Reply:
It's a bit off-topic for this PR, but this timeout is too high anyway.

Reply:
Right it was too high in the first place, and since I've also decreased the number of requested blocks the timeout should be readjusted.